commands/audit/audit.go

```diff
@@ -144,13 +197,10 @@ func (auditCmd *AuditCommand) Run() (err error) {
 		messages = []string{coreutils.PrintTitle("The ‘jf audit’ command also supports JFrog Advanced Security features, such as 'Contextual Analysis', 'Secret Detection', 'IaC Scan' and ‘SAST’.\nThis feature isn't enabled on your system. Read more - ") + coreutils.PrintLink(utils.JasInfoURL)}
 	}
 	if err = output.NewResultsWriter(auditResults).
-		SetHasViolationContext(auditCmd.HasViolationContext()).
```

Comment:
Question: er set the Violation context in auditParams, but how does it get passed to the resultsWriter so we know we are in a violation context?

Reply:
based on the value in the results - I added `ResultsContext` struct for this (there is a way to override violation context for build scan only at result writer as well)